### ISSUE TITLE
[accessibility] Html5 element <footer> used in dashboard page does not have the corresponding WAI-ARIA "contentinfo" role defined.

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/gingerbase#82]
Html5 element <footer> used in the content to identify a page Landmark/Section does not have the corresponding WAI-ARIA "contentinfo" role defined.
